full_snapshot.proto

```diff
@@ -396,6 +396,7 @@ message FunctionInformation {
   bool strict = 13;
   bool returns_set = 14;
   string volatile = 15;
+  bool procedure = 16;
```

Comment:
```
enum FunctionType {
  UNKNOWN = 0;
  FUNCTION = 1;
  PROCEDURE = 2;
  AGGREGATE = 3;
  WINDOW = 4;
}
```

=> On read, if UNKNOWN, look at `aggregate` and `window` fields - otherwise ignore those fields and use type only

Reply:
I used `FunctionKind` instead of `FunctionType` to align with `prokind` in the catalog